src/test/java/org/fcrepo/camel/FcrepoProducerTest.java

```diff
@@ -707,58 +526,6 @@ public void testGetSecureProducer() throws Exception {
     }
 
     @Test
-    public void testGetSecureProducerWithScheme() throws Exception {
```

Comment:
This method seems unrelated to `transform` and `tombstone`. Was it intentionally removed?